wolframalpha/__init__.py

```diff
@@ -1,84 +1,184 @@
-from xml.etree import ElementTree as etree
 from six.moves import urllib
+import xmltodict
 
 from . import compat
 
 compat.fix_HTTPMessage()
 
+
+class Client(object):
+    """
+    Wolfram|Alpha v2.0 client
+    
+    Pass an ID to the object upon instantiation, then
+    query Wolfram Alpha using the query method.
+    """
+    def __init__(self, app_id='Q59EW4-7K8AHE858R'):
```

Comment:
Oh, sneaky. By moving this code, I didn't see that this app_id default got set. This definitely needs to go. It was intentionally removed from the code in 648a4b5aac41f9.